src/stream-to-value.js

```
@@ -0,0 +1,12 @@
+'use strict'
+
+const concat = require('concat-stream')
+
+/*
+  Concatenate a stream to a single value.
+*/
+function streamToValue (res, callback) {
+  res.pipe(concat((data) => callback(null, data)))
+}
+
+module.exports = streamToValue
```

In case of error on the stream (connection dropped) this fails silently